### DISCUSSION TITLE
Ground binding acting up again

### DISCUSSION THREAD

**Discussion by Annolago** [flybywiresim/aircraft#4089] [General]
Hi

Is it just me or has the ground resistance gone haywire again? Update to latest version of experimental version with the custom FBW (545c472) and in idle I achieved almost 50 in groundspeed. The weight of the plane was about 62 tons, so close to the max 65 for touch and go.
thx

**Comment by hotshotp** [comment]
The behavior you are describing is perfectly normal and intentional. IRL pilots use brakes when they taxi because the plane accelerates on idle

**Reply by Annolago** [reply]
Yes, I am aware but the plane was acting like on ice. Normally it will need a gentle nudge to start moving, mine was accelerating like a race car..
Will do a clean install and see if the problem is still there..

**Comment by Annolago** [comment]
So found the issue. For some reason the FlyByWire folder changed names from A32NX to "flybywire-aircraft-a320-neo" and that means that the throttle config that was still sitting in the old folder is no longer working. The throttle ini needs to be in a subfolder with the same name. 
So my TM TCA Quadrant had an idle at almost 40% thrust - and no reverse. No wonder I was almost flying on the ground. 
So heads up everyone, if you are having issues then check to see if the install folder is still what you think it is..
